pkg/minishift/registration/redhat_test.go

```diff
@@ -27,22 +27,18 @@ import (
 
 var (
 	param = &RegistrationParameters{
-		Username:        "foo",
-		Password:        "foo",
```

Comment:
Actually, I don't think we should remove this. We should keep this in place. The whole idea is/was to allow other registrars. It was always the intention to pass proxy variables viaa RegistrationParameters. If we remove these parameters, then we are again making things Red Hat subscription manager specific. 

To stick with our initial intention, we should keep this (and everything related to it) and just not make use of it in redhat.go.